visgraph/graphcore.py

```diff
@@ -303,6 +303,14 @@ def delNodeProp(self, node, prop):
                 self.nodeprops[prop].pop(pval,None)
         return pval
 
+    def delNodesProps(graph, props=('up','down')):
```

Comment:
please make the self reference named correctly...

These seem like use-case specific defaults in what should be a general purpose API.  Should probably be:

```python
def delNodesProps(self, props):
```